frontend/src/theme/components/Tooltip.ts

```diff
@@ -0,0 +1,26 @@
+import { ComponentMultiStyleConfig } from '@chakra-ui/theme'
+
+export const TOOLTIP_THEME_KEY = 'CustomTooltip'
+export const CustomTooltip: ComponentMultiStyleConfig = {
+  parts: ['tooltip', 'wrapper'],
+  baseStyle: {
+    tooltip: {
+      '--tooltip-bg': 'var(--chakra-colors-secondary-700)',
+      px: '0.75rem',
+      py: '0.5rem',
+      color: 'white',
+      textStyle: 'body-2',
+      borderRadius: '0.25rem',
+      textAlign: 'left',
+      margin: '0.25rem',
+      maxWidth: '19.5rem',
```

Comment:
Is there a need for maxWidth?

Reply:
The default tooltip also has maxWidth, so I asked Pearly about this and she said we could use 312px. I think this is to prevent the tooltip from becoming too big.